prime-angular-frontend/src/app/modules/paper-enrolment/pages/overview-page/overview-page.component.html

```diff
@@ -13,254 +13,28 @@
-              (click)="onRoute(PaperEnrolmentRoutes.OBO_SITES)">
-        <mat-icon>edit</mat-icon>
-      </button>
-    </app-page-subheader>
-
-    <ng-container *ngFor="let careSetting of enrollee.enrolleeCareSettings">
-
-      <ng-container *ngFor="let oboSite of enrollee.oboSites">
-        <ng-container *ngIf="oboSite.careSettingCode === careSetting.careSettingCode">
-          <app-enrollee-property title="Care Setting"
-                                 [makeBold]="true">
-            <div class="mb-3">{{ careSetting.careSettingCode | configCode: 'careSettings' }}
-              <span *ngIf="oboSite.careSettingCode === CareSettingEnum.HEALTH_AUTHORITY">
-                ({{ oboSite.healthAuthorityCode | configCode: 'healthAuthorities' | capitalize: true }})
-              </span>
-            </div>
-
-            <app-enrollee-property *ngIf="oboSite.careSettingCode !== CareSettingEnum.HEALTH_AUTHORITY"
-                                   title="Site Name"
-                                   [makeBold]="true">
-              {{ oboSite.siteName | default }}
-            </app-enrollee-property>
-
-            <app-enrollee-property *ngIf="oboSite.careSettingCode === CareSettingEnum.HEALTH_AUTHORITY"
-                                   title="Facility Name"
-                                   [makeBold]="true">
-              {{ oboSite.facilityName | default }}
-            </app-enrollee-property>
-
-            <app-enrollee-property *ngIf="oboSite.careSettingCode !== CareSettingEnum.HEALTH_AUTHORITY"
-                                   title="Site ID"
-                                   [makeBold]="true">
-              {{ oboSite.pec | default }}
-            </app-enrollee-property>
-
-            <app-enrollee-property title="Job Title"
-                                   [makeBold]="true">
-              {{ oboSite.jobTitle | default }}
-            </app-enrollee-property>
-
-            <app-enrollee-property title="Site Address"
-                                   [makeBold]="true">
-              <app-enrollee-property title="Street">
-                {{ oboSite.physicalAddress?.street | default }}
-              </app-enrollee-property>
-
-              <app-enrollee-property title="City">
-                {{ oboSite.physicalAddress?.city | default }}
-              </app-enrollee-property>
-
-              <app-enrollee-property title="Province">
-                {{ oboSite.physicalAddress?.provinceCode | configCode: 'provinces' | default }}
-              </app-enrollee-property>
-
-              <app-enrollee-property title="Postal Code">
-                {{ oboSite.physicalAddress?.postal | postal | default }}
-              </app-enrollee-property>
-
-            </app-enrollee-property>
-          </app-enrollee-property>
-        </ng-container>
-      </ng-container>
-    </ng-container>
-
-  </app-page-section>
+  <app-demographic-overview [firstName]="enrollee?.firstName"
```

Comment:
You'll want to refactor and follow the example I have in HealthAuthorities.  Passing in this many bindings is too much.  When I see bindings like this especially from the same model it indicates the components needs a bit of a refactor.